controllers/sms-gateway.js

```diff
@@ -3,25 +3,27 @@
  * @see https://github.com/medic/medic-gateway
  */
 
-var async = require('async'),
-    messageUtils = require('./message-utils'),
-    recordUtils = require('./record-utils'),
-
-    // map from the medic-gateway state to the medic-webapp state
-    STATUS_MAP = {
-      UNSENT: 'received-by-gateway',
-      PENDING: 'forwarded-by-gateway',
-      SENT: 'sent',
-      DELIVERED: 'delivered',
-      FAILED: 'failed',
-    };
+const _ = require('underscore'),
+      async = require('async'),
+      messageUtils = require('./message-utils'),
+      recordUtils = require('./record-utils'),
+
+      // map from the medic-gateway state to the medic-webapp state
+      STATUS_MAP = {
+        UNSENT: 'received-by-gateway',
+        PENDING: 'forwarded-by-gateway',
+        SENT: 'sent',
+        DELIVERED: 'delivered',
+        FAILED: 'failed',
+      };
 
 function warn() {
-  var args = Array.prototype.slice.call(arguments, 0);
+  const args = Array.prototype.slice.call(arguments, 0);
   args.unshift('WARN', 'sms-gateway');
   console.error.apply(console, args);
 }
 
+// TODO: pull the logic out of CouchDB, make local, make bulk update
 function saveToDb(message, callback) {
   recordUtils.createByForm({
```

Comment:
Yeah I don't get why it's by form either. This name should get refactored out once we've made it a bulk update, so I'm OK with it hanging around a little longer.